src/zenml/model/model_version.py

```diff
@@ -470,6 +496,25 @@ def _get_model_version(self) -> "ModelVersionResponse":
         if not self._id:
             self._id = mv.id
 
+        difference: Dict[str, Any] = {}
+        if mv.description != self.description:
+            difference["description"] = {
+                "config": self.description,
+                "db": mv.description,
+            }
+        configured_tags = set(self.tags or [])
+        db_tags = {t.name for t in mv.tags}
+        if db_tags != configured_tags:
+            difference["tags added"] = list(configured_tags - db_tags)
+            difference["tags removed"] = list(db_tags - configured_tags)
+        if difference:
+            logger.warning(
+                "Provided model version configuration does not match existing model "
+                f"version `{self.name}::{self.version}` with the following "
+                f"changes: {difference}. If you want to update the model version "
```

Comment:
Will this display nicely? I guess it's just logged but wondering if we can make it a bit more readable?

Reply:
Was thinking to leave a linebreak at least, but if you think it looks ok then that's good enough for me. It's logging output anyway, not our CLI output etc.